### PR TITLE
Docs: Add JSON format for webhook notifications to differences

### PR DESCRIPTION
**Pull request by grobinson-grafana**
**What is this feature?**

Updates the **Differences and limitations** page:

1. Adds text about JSON format for webhook notifications
2. Removes text about pausing alerts as these are supported
3. Fixes numbered list being reset

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by yuri-tceretian** [comment]
The change LGTM. However, I wonder if it would make sense to create a separate page and provide two formats side by side.

File: docs/sources/alerting/migrating-alerts/migrating-legacy-alerts.md [hunk 1]
```diff
@@ -20,16 +20,17 @@ longer supported. We refer to these as [Differences]({{< relref "#differences" >
...
 
 4. Notification channels are migrated to an Alertmanager configuration with the appropriate routes and receivers. Default notification channels are added as contact points to the default route. Notification channels not associated with any Dashboard alert go to the `autogen-unlinked-channel-recv` route.
 
 5. Unlike legacy dashboard alerts where images in notifications are enabled per contact point, images in notifications for Grafana Alerting must be enabled in the Grafana configuration, either in the configuration file or environment variables, and are enabled for either all or no contact points. Refer to [images in notifications](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/images-in-notifications/).
-6. Grafana Alerting does not support pausing the evaluation of alert rules. After migration, all paused alert rules will become active, which may cause unexpected notifications to be sent.
+
+6. The JSON format for webhook notifications has changed in Grafana Alerting, using that of [Prometheus Alertmanager](https://prometheus.io/docs/alerting/latest/configuration/#webhook_config).
```
Comment (brendamuir) [inline docs/sources/alerting/migrating-alerts/migrating-legacy-alerts.md:33]:
```suggestion
6. The JSON format for webhook notifications has changed in Grafana Alerting and uses the format from[Prometheus Alertmanager](https://prometheus.io/docs/alerting/latest/configuration/#webhook_config).
```

**Review by brendamuir — APPROVED** [review]

File: docs/sources/alerting/migrating-alerts/migrating-legacy-alerts.md [hunk 2]
```diff
@@ -20,16 +20,17 @@ longer supported. We refer to these as [Differences]({{< relref "#differences" >
 
 2. Read and write access to legacy dashboard alerts and Grafana alerts are governed by the permissions of the folders storing them. During migration, legacy dashboard alert permissions are matched to the new rules permissions as follows:
 
-- If alert's dashboard has permissions, it will create a folder named like `Migrated {"dashboardUid": "UID", "panelId": 1, "alertId": 1}` to match permissions of the dashboard (including the inherited permissions from the folder).
-- If there are no dashboard permissions and the dashboard is under a folder, then the rule is linked to this folder and inherits its permissions.
-- If there are no dashboard permissions and the dashboard is under the General folder, then the rule is linked to the `General Alerting` folder, and the rule inherits the default permissions.
+   - If there are dashboard permissions,a folder named `Migrated {"dashboardUid": "UID", "panelId": 1, "alertId": 1}` is created to match the permissions of the dashboard (including the inherited permissions from the folder).
```
Comment (grobinson-grafana) [inline docs/sources/alerting/migrating-alerts/migrating-legacy-alerts.md:23]:
```suggestion
   - If there are dashboard permissions, a folder named `Migrated {"dashboardUid": "UID", "panelId": 1, "alertId": 1}` is created to match the permissions of the dashboard (including the inherited permissions from the folder).
```

File: docs/sources/alerting/migrating-alerts/migrating-legacy-alerts.md [hunk 3]
```diff
@@ -32,8 +32,9 @@ longer supported. We refer to these as [Differences]({{< relref "#differences" >
...
 
-5. Unlike legacy dashboard alerts where images in notifications are enabled per contact point, images in notifications for Grafana Alerting must be enabled in the Grafana configuration, either in the configuration file or environment variables, and are enabled for either all or no contact points. Refer to [images in notifications]({{< relref "../manage-notifications/images-in-notifications" >}}).
-6. Grafana Alerting does not support pausing the evaluation of alert rules. After migration, all paused alert rules will become active, which may cause unexpected notifications to be sent.
+5. Unlike legacy dashboard alerts where images in notifications are enabled per contact point, images in notifications for Grafana Alerting must be enabled in the Grafana configuration, either in the configuration file or environment variables, and are enabled for either all or no contact points. Refer to [images in notifications](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/images-in-notifications/).
```
Comment (grobinson-grafana) [inline docs/sources/alerting/migrating-alerts/migrating-legacy-alerts.md:35]:
```suggestion
5. Unlike legacy dashboard alerts where images in notifications are enabled per contact point, images in notifications for Grafana Alerting must be enabled in the Grafana configuration, either in the configuration file or environment variables, and are enabled for either all or no contact points. Refer to [images in notifications]({{< relref "../manage-notifications/images-in-notifications" >}}).
```

**Pull request merged**